src/engine/controls/loopingcontrol.cpp

```diff
@@ -722,6 +762,33 @@ void LoopingControl::slotLoopExit(double val) {
     }
 }
 
+void LoopingControl::slotLoopToggle(double val) {
+    if (!m_pTrack || val <= 0.0) {
+        return;
+    }
+
+    // If we're looping, stop looping
+    if (m_bLoopingEnabled) {
+        // If loop roll was active, also disable slip.
+        if (m_bLoopRollActive) {
+            m_pSlipEnabled->set(0);
+            m_bLoopRollActive = false;
+            m_activeLoopRolls.clear();
+        }
+        setLoopingEnabled(false);
+        //qDebug() << "loop_toggle looping off";
+    } else {
+        // If we're not looping, enable the loop.
+        // In contrast to the reloop_toggle CO, we do not jump in any case.
```

Comment:
I do not understand the use case for this new Control. Why would you enable a loop that is behind the playhead but not jump back to it?

Reply:
This can make sense if you want to enable the loop in advance, then jump to a hotcue that is in front of the loop.
This is how my controller behaves in Serato, and I'm trying to mirror that.

Reply:
I do not think we should introduce such a similar Control as reloop_toggle. People mapping controllers will not know which one to map. Let's either change the behavior of reloop_toggle or not.

I think it could make sense to change the behavior of reloop_toggle. If reloop_toggle did not jump back to the loop, previously, you would have needed to also set a hotcue at the beginning of the loop to go back to it. However, with this new feature you can save the loop to a hotcue and use hotcue_X_activate to do the same thing. What does this change in behavior allow? You could, for example, set a different hotcue on a lead in beat before the loop and press reloop_toggle before that hotcue without having to be so careful that you press reloop_toggle before the end of the loop passes.

Reply:
ping

Reply:
The problem is: `reloop_toggle` has `reloop` in its name. So if we don't jump back to the beginning of the loop, this would be a misnomer and also confusing IMHO. Hence, I'd rather add a new CO instead.

Reply:
I still disagree. There is no need for a new CO.

Reply:
Let's discuss this on Zulip: https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/Using.20reloop_toggle.20CO.20for.20toggling.20a.20loop.20without.20jumping

Reply:
Ping. I still do not agree there is a good reason to add this confusing new CO.

Reply:
I'd still like to have this option, but I can look into making the `loop_enabled` CO read-write instead (it's currently RO). Then we don't need a new CO and I can still map my controller as labeled.

Reply:
Done, the CO has been removed: https://github.com/mixxxdj/mixxx/pull/2194/commits/a2c7c0e4b4c884129dafe457ccb597afb0230aff